pkg/config/config.go

```diff
@@ -697,8 +697,20 @@ func InitConfig(config Config) {
 	config.SetKnown("apm_config.replace_tags")
 	config.SetKnown("apm_config.obfuscation.elasticsearch.enabled")
 	config.SetKnown("apm_config.obfuscation.elasticsearch.keep_values")
+	config.SetKnown("apm_config.obfuscation.elasticsearch.transform_values")
+	config.SetKnown("apm_config.obfuscation.elasticsearch.transformer_type")
 	config.SetKnown("apm_config.obfuscation.mongodb.enabled")
 	config.SetKnown("apm_config.obfuscation.mongodb.keep_values")
+	config.SetKnown("apm_config.obfuscation.mongodb.transform_values")
+	config.SetKnown("apm_config.obfuscation.mongodb.transformer_type")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan.enabled")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan.keep_values")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan.transform_values")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan.transformer_type")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan_normalize.enabled")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan_normalize.keep_values")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan_normalize.transform_values")
+	config.SetKnown("apm_config.obfuscation.sql_exec_plan_normalize.transformer_type")
```

Comment:
Wouldn't it be better to group these under `apm_config.obfuscation.sql.*`? I see some other PRs attempting to add that and we may want more options later on. One possibility is `apm_config.obfuscation.sql.exec_plan.*`

Reply:
I made it a bit simpler in https://github.com/DataDog/datadog-agent/commit/c5014db632ce98ccd97685e861f10007c58222dd 

We can't easily group them because each of these `(es, mongo, sql_exec_plan, and sql_exec_plan_normalize)` are configuring an independent json obfuscator. They can each have different keep_values and `obfuscate_sql_values`.